pkg/proxy/nftables/proxier.go

```diff
@@ -1136,6 +1137,18 @@ func (s *nftElementStorage) cleanupLeftoverKeys(tx *knftables.Transaction) {
 	s.resetLeftoverKeys()
 }
 
+// knftables doesn't supporting listing the full table yet, this function is a workaround.
+func listTable() (string, error) {
+	nftPath, err := exec.LookPath("nft")
+	if err != nil {
+		return "", fmt.Errorf("could not find nftables binary: %w", err)
+	}
```

Comment:
you don't actually need to do this separately, if you don't care about having a separate error message for this case; you can just pass `"nft"` to `exec.Command`.